avogadro/core/molecule.cpp

```diff
@@ -803,7 +803,7 @@ void Molecule::perceiveBondsSimple(const double tolerance, const double min)
   // find molecule bounding box
   Vector3 min_pos = m_positions3d[0];
   Vector3 max_pos = m_positions3d[0];
-  for (Index i = 1; i < atomCount(); i++) {
+  for (Index i = 0; i < atomCount(); i++) {
```

Comment:
You don't need to start the loop at 0 because you already grabbed the position of atom zero in the lines above.